cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -2132,6 +2132,7 @@ describe('CourseOutlinePage', function() {
                 ]);
                 createCourseOutlinePage(this, mockCourseJSON, false);
                 setSelfPacedCustomPLS();
+                course.set('start', '2014-07-09T00:00:00Z');
```

Comment:
So instead of doing this, you can just pass his into the first argument to `createMockCourseJSON` a few lines up.

Reply:
yeaa that was my initial attempt but it didn't work for some reason:( so manually setting it like this seems to work and not sure what the difference is